Yauheni_Mandrik_HTML_CSS/styles/media.css

```diff
@@ -0,0 +1,511 @@
+
+
+@media (max-width:1439px) {
+    .product {
+      
+        grid-template-columns: repeat(2, 1fr);
+        grid-template-rows: min-content min-content auto auto auto;
+        
+        gap: 20px;
+        
+    }
+    .product h3{
+        grid-column: 1/3;
+        grid-row: 1;
+    }
+    .product h2{
+        grid-column: 1/3;
+        grid-row: 2;
+    }
+    
+    .product_item:nth-of-type(1){
+        grid-row: 3;
+        grid-column: 1;
+    }
+    
+    .product_item:nth-of-type(2){
+        grid-row: 3;
+        grid-column: 2;
+    }
+    
+    .product_item:nth-of-type(3){
+        grid-row: 4;
+        grid-column: 1;
+    }
+    .product_item:nth-of-type(4){
+        grid-row: 4;
+        grid-column: 2;
+    }
+    .product_item:nth-of-type(5){
+        grid-row: 5;
+        grid-column: 1;
+    }
+    .product_item:nth-of-type(6){
+        grid-row: 5;
+        grid-column: 2;
+    }
+
+    .offer_products_wrapper {
+        grid-template-columns: repeat(2, 1fr);
+        grid-template-rows: repeat(6, auto);
+    }
+
+    .offer_products_wrapper h3 {
+        grid-row: 1;
+        grid-column: 1/2;
+        
+    }
+    
+    .offer_products_wrapper h2{
+        grid-row: 2;
+        grid-column: 1/2;
+        
+    }
+
+    .offer_products_wrapper .all_products{
+        grid-row: 2;
+        grid-column: 2/3;
+        
+    }
+    
+    .testimonial {
+        background-image: none;
+    }
+
+
+
+}
+
+@media (max-width:1280px) {
+    .promo {
+        padding: 100px 0;
+        
+    }
+    .offer {
+        padding: 70px 0;
+    }
+
+    .about_wrapper {
+        padding: 65px 0;
+    }
+
+    .product {
+        padding: 60px 0;
+    }
+
+    .testimonial {
+        padding: 65px 0;
+    }
+    .offer_products {
+        padding: 65px 0;
+    }
+    .offer_products_wrapper {
+        padding: 0;
+    }
+
+    .types_of_products {
+        padding: 70px 0;
+    }
+}
+
+
+
+
+@media (max-width:1024px) {
+    .promo {
+        padding: 50px 0;
+        
+    }
+    .offer {
+        padding: 35px 0;
+    }
+
+    .about_wrapper {
+        padding: 32px 0;
+    }
+
+    .product {
+        padding: 30px 0;
+    }
+
+    .testimonial {
+        padding: 32px 0;
+    }
+
+    .testimonial_top img {
+        margin: 0;
+    }
+
+    
+
+    .testimonial_top p:nth-of-type(1),
+    .testimonial_top p:last-of-type {
+        margin-bottom:10px;
+    }
+
+    .testimonial_bot {
+        margin-top: 30px;
+    }
+
+   
+
+
+    .offer_products {
+        padding: 32px 0;
+    }
+    .offer_products_wrapper {
+        padding: 0;
+    }
+
+    .types_of_products {
+        padding: 35px 0;
+    }
+}
+
+
+
+@media (max-width:1096px) {
+
+    .subscribe_wrapper {
+        background-image: none;
```

Comment:
return background-image: url(../img/subscribe_bg.png); should be simalary how in design
try for subscribe_form min-height and add padding for this or parent div. h2 should has little size